filters/contains-non-ascii/README.md

```diff
@@ -0,0 +1,12 @@
+## unicode filter
+
+## What type of a filter is this?
+
+This filter filters examples which contain non-ascii characters.
+Author: Jan Pfister
+
+## Why is measuring performance on this split important?
+This filter can be used to find datapoints containing non-ascii unicode characters. Filtering out and testing on examples containing these characters can provide feedback for improving models accordingly as most models are trained on plain english text mostly containing ascii characters. Sometimes non-ascii character are even explicitly stripped away.
```

Comment:
nit: English, not english

Also, since this works for all languages, maybe change this paragraph to address what it would do for other languages?